lib/restbase.js

```diff
@@ -325,13 +325,20 @@ RESTBase.prototype._request = function(req) {
         });
 
         if (accessRestricted) {
-            return P.all([
-                reqHandlerPromise,
-                childRESTBase._authService.checkPermissions(childRESTBase, childReq)
-            ])
-            .then(function(res) {
-                return res[0];
-            });
+            if (req.method === 'get' || req.method === 'head') {
```

Comment:
I realise this is a general *check permissions* routine, but for `POST`s I think we should separate that.